src/platforms/platform.dom.js

```diff
@@ -312,29 +313,31 @@ function injectCSS(rootNode, css) {
 	}
 }
 
-export default {
-	/**
-	 * When `true`, prevents the automatic injection of the stylesheet required to
-	 * correctly detect when the chart is added to the DOM and then resized. This
-	 * switch has been added to allow external stylesheet (`dist/Chart(.min)?.js`)
-	 * to be manually imported to make this library compatible with any CSP.
-	 * See https://github.com/chartjs/Chart.js/issues/5208
-	 */
-	disableCSSInjection: false,
-
-	/**
-	 * This property holds whether this platform is enabled for the current environment.
-	 * Currently used by platform.js to select the proper implementation.
-	 * @private
-	 */
-	_enabled: typeof window !== 'undefined' && typeof document !== 'undefined',
+/**
+ * Platform class for charts that can access the DOM and global window/document properties
+ * @constructor
```

Comment:
It seems weird to me to put the `@constructor` tag on the class instead of the constructor

Reply:
whoops, meant to move that to the constructor